spiel/cmd.c

```diff
@@ -1666,18 +1666,37 @@ struct m0_proc_data {
 	struct m0_spiel_bckey pd_bckey;
 	struct m0_spiel_bcrec pd_bcrec;
 	struct m0_tlink       pd_link;
+	uint64_t              pd_magic;
```

Comment:
m0_proc_data::pd_magic will be used when the list spiel_proc_counter_item::sci_counters is implemented for retry mechanism.